lte/gateway/python/integ_tests/federated_tests/docker/run.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+import os
+import subprocess
+
+SNOWFLAKE_PATH = '../../../../../../.cache/feg/'
+SNOWFLAKE_FILE = '../../../../../../.cache/feg/snowflake'
```

Comment:
Can we replace  `../../../../../..` with MAGMA_ROOT?
Something like `os.environ["MAGMA_ROOT"]`

Reply:
Good idea, I changed the code accordingly.